### PR TITLE
Loglylift Bid Adapter: add new bid adapter

### PR DESCRIPTION
**Pull request by HashimotoLogly**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: 'loglylift',
  params: {
    adspotId: 4302078
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer : dev@logly.co.jp
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@HashimotoLogly can you please pass in advertiserdomain. That is a requirement now for new adapters https://github.com/prebid/Prebid.js/issues/6466

**Comment by HashimotoLogly** [comment]
@ChrisHuie 
Hi. Thank you for your review!
Our adapter do not intent to support `advertiserDomains`.  And according to the documentation, it says `advertiserDomains ` is optional.
Which is correct, optional or required?
Banners_and_Alerts_と_How_to_Add_a_New_Prebid_js_Bidder_Adapter_for_Header_Bidding

**Comment by HashimotoLogly** [comment]
@bretg @ChrisHuie 
Ok,I understood and added the code for `advertiserDomains`. Could you check again?

File: test/spec/modules/loglyliftBidAdapter_spec.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+      const data = JSON.parse(request.data);
+      expect(data.auctionId).to.equal(nativeBidRequests[0].auctionId);
+      expect(data.bidderRequestId).to.equal(nativeBidRequests[0].bidderRequestId);
+      expect(data.transactionId).to.equal(nativeBidRequests[0].transactionId);
+      expect(data.adUnitCode).to.equal(nativeBidRequests[0].adUnitCode);
+      expect(data.bidId).to.equal(nativeBidRequests[0].bidId);
+      expect(data.mediaTypes).to.deep.equal(nativeBidRequests[0].mediaTypes);
+      expect(data.params).to.deep.equal(nativeBidRequests[0].params);
+      expect(data.prebidJsVersion).to.equal('6.5.0-pre');
```
Comment (robertrmartinez) [inline test/spec/modules/loglyliftBidAdapter_spec.js:112]: This obviously makes tests fail as we move through versions.

https://github.com/prebid/Prebid.js/pull/7869

@HashimotoLogly @ChrisHuie

**Pull request merged**